google/ggrc-core
# GGRC-5642 Workflow admin is missed in new workflow popup after clicking "save and add another"

# Issue description

Workflow admin is missed in new workflow popup after clicking "save and add another".

# Steps to test the changes

1. Invoke new workflow popup
2. Fill out required fields and click "save and ad another" button
3. Look at Admin field once new workflow popup is displayed: is missed
4. Fill title and look at the save buttons: active 

*Actual Result:* Workflow 'admin' is missed in new workflow popup after clicking "save and add another"
*Expected Result:* Workflow admin should not be missed in new workflow popup after clicking "save and add another"

# Solution description

Was removed potential bug (`isNewInstance` logic).
The main issue was appeared due to WF's modal template doesn't re-render inner content like other templates via `{{#instance}}{{instance}}` construction. Due to this reason, we should check the moment, when `instance` for modal was changed. When such moment was detected, WF's roles must be pre populated.


# Sanity checklist




# PR Review checklist
